### PR TITLE
Standardise output of "airflow connections export" to those of "airflow variables export" and "airflow users export"

### LINKED ISSUE #34232: Standardise output of "airflow connections export" to those of "airflow variables export" and "airflow users export"
### Description

The three commands:

```
              airflow users export /opt/airflow/dags/users.json &&
              airflow variables export /opt/airflow/dags/variables.json &&
              airflow connections export /opt/airflow/dags/connections.json
```

give back:

```
7 users successfully exported to /opt/airflow/dags/users.json
36 variables successfully exported to /opt/airflow/dags/variables.json
Connections successfully exported to /opt/airflow/dags/connections.json.
```

### Use case/motivation

Standardise output of "airflow connections export" to those of "airflow variables export" and "airflow users export"

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Jayden-Chiu**
Output should now be standardized to something similar to users and variables
```
7 connections successfully exported to /opt/airflow/dags/connections.json.
```
Closes: #34232 

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by uranusjr — APPROVED** [review]

**Review by Lee-W — APPROVED** [review]

File: airflow/cli/commands/connection_command.py [hunk 1]
```diff
@@ -197,9 +197,9 @@ def connections_export(args):
         f.write(msg)
 
     if file_is_stdout:
-        print("\nConnections successfully exported.", file=sys.stderr)
+        print(f"\n{len(connections)} connections successfully exported.", file=sys.stderr)
```
Comment (utkarsharma2) [inline airflow/cli/commands/connection_command.py:200]: @Jayden-Chiu Not a blocker, How can we ensure that the output of `export connection` `export variable` and `users export` doesn't mismatch by any further changes in the codebase?

Can we add a single function to prepare a similar output?

Comment (hussein-awala) [reply airflow/cli/commands/connection_command.py:200]: @utkarsharma2 good point, could you open a new PR to do that?

**Review by hussein-awala — APPROVED** [review]

**Comment by hussein-awala** [comment]
Congrats on your first commit :tada:

File: airflow/cli/commands/connection_command.py (hunk 1 above)
Comment (utkarsharma2) [reply airflow/cli/commands/connection_command.py:200]: @hussein-awala Sure, I'll raise that.

**Pull request merged**